main.swift

```diff
@@ -361,9 +361,22 @@ do {
     }
     
     try performCommand(description: "Copying template folder") {
+        let ignorableItems = ["readme.md", "license"]
+        let ignoredItems = try fileManager.contentsOfDirectory(atPath: destination).map {
```

Comment:
This builds the list of the files that won't be copied by checking which ignorable items are contained in the destination folder.

I thought it was better (and faster) to build the list once and compare every `itemName` against it rather than checking if `itemName` is ignorable and then calling `fileExists(atPath:)`.